tests/conftest.py

```diff
@@ -10,24 +10,47 @@ def basic_crux_df():
     """A simple crux-like dataframe"""
     df = pd.DataFrame(
         [
-            [1, 10, 0.7, "A", "target", 0.7],
-            [2, 20, 0.4, "B", "decoy", 0.1],
-            [3, 30, 0.1, "C", "target", 0.2],
-            [4, 40, 0.55, "D", "target", 0.8],
-            [5, 50, 0.25, "E", "target", 0.25],
-            [1, 10, 0.6, "F", "target", 0.6],
-            [2, 20, 0.2, "G", "decoy", 0.2],
-            [3, 30, 0.7, "H", "decoy", 0.4],
-            [4, 40, 0.56, "I", "target", 0.56],
-            [5, 50, 0.3, "J", "decoy", 0.3],
+            ["file1", 1, 10, 0.7, "APPLE", "target", 0.7, "prot1", "APPLE"],
+            ["file1", 2, 20, 0.4, "ANANAB", "decoy", 0.1, "prot2", "BANANA"],
+            ["file1", 3, 30, 0.1, "CHERRY", "target", 0.2, "prot3", "CHERRY"],
+            ["file1", 4, 40, 0.55, "DURIAN", "target", 0.8, "prot4", "DURIAN"],
+            [
+                "file1",
+                5,
+                50,
+                0.25,
+                "EGGPLANT",
+                "target",
+                0.25,
+                "prot5",
+                "EGGPLANT",
+            ],
```

Comment:
**nitpik:** It would be nice if this line could be shortened to fit on a single line. Maybe use `f1` for the file name?